labelbox/schema/model_run.py

```diff
@@ -262,6 +262,56 @@ def update_status(self,
             },
             experimental=True)
 
+    @experimental
+    def update_config(self, config: Dict[str, Any]) -> Dict[str, Any]:
+        """ 
+         Updates the model run's training metadata config
```

Comment:
Sorry to be a stickler, but could we capitalize `Model Run` everywhere in the comments? Just wanna make sure the docs are formalized, since they'll be public facing. Thanks!